allauth/socialaccount/providers/dingtalk/client.py

```diff
@@ -0,0 +1,33 @@
+import requests
+
+from allauth.socialaccount.providers.oauth2.client import (
+    OAuth2Client,
+    OAuth2Error,
+)
+
+
+class DingtalkOAuth2Client(OAuth2Client):
+    def get_access_token(self, code):
+        data = {
+            "clientId": self.consumer_key,
+            "clientSecret": self.consumer_secret,
+            "code": code,
+            "grantType": "authorization_code",
+        }
+        params = None
+        self._strip_empty_keys(data)
+        url = self.access_token_url
+        if self.access_token_method == "GET":
+            params = data
+            data = None
+        resp = requests.request(self.access_token_method, url, params=params, json=data)
+        access_token = None
```

Comment:
**Comment**: It would be good to add a `resp.raise_for_status()` here.

Reply:
thx, i have added it.